### PR TITLE
Added Nav Bar Slide over

### PR DESCRIPTION
**Pull request by shawnpatel96**
Added a Slide over event upon mouse over for the Navigation Bar

### CODE REVIEW

File: src/styles/css/main.css [hunk 1]
```diff
@@ -1239,27 +1239,47 @@ p {
...
   height: 50px;
-  margin-left: 160px;
 }
 .leaderboardimg {
   filter: invert(1);
   width: 50x;
   height: 50px;
-  margin-left: 160px;
+}
+.logoutimage {
```
Comment (brandleyj) [inline src/styles/css/main.css]: Needs margin right of the image.

**Review by brandleyj — CHANGES_REQUESTED** [review]
Needs margin on the right of the logout button.

File: src/styles/css/main.css (hunk 1 above)
Comment (shawnpatel96) [reply src/styles/css/main.css]: Will change it now!

**Review by brandleyj — APPROVED** [review]
Changed the margin as requested. We're all good here now

**Pull request merged**